editoast/src/views/v2/path/pathfinding.rs

```diff
@@ -174,6 +114,7 @@ async fn pathfinding_blocks(
     let result: Option<PathfindingResult> =
         redis_conn.json_get_ex(&hash, CACHE_PATH_EXPIRATION).await?;
     if let Some(pathfinding) = result {
+        info!("Pathfinding call: hit cache");
```

Comment:
I'm wondering if you need the `Pathfinding call` prefix here. The log statement will contain metadata for the module and the file (and line) of where this log event happened, which should be enough for context.

Reply:
Great